twisted/internet/defer.py

```diff
@@ -909,16 +910,16 @@ def gatherResults(deferredList, consumeErrors=False):
     This differs from L{DeferredList} in that you don't need to parse
     the result for success/failure.
 
-    @type deferredList:  C{list} of L{Deferred}s
+    @type deferredList:  L{list} of L{Deferred}s
 
     @param consumeErrors: (keyword param) a flag, defaulting to False,
-        indicating that failures in any of the given L{Deferreds} should not be
-        propagated to errbacks added to the individual L{Deferreds} after this
+        indicating that failures in any of the given L{Deferred}s should not be
+        propagated to errbacks added to the individual L{Deferred}s after this
         L{gatherResults} invocation.  Any such errors in the individual
```

Comment:
Let's just leave it for now, and get this in.
There are other epydoc errors in the tree to focus on. :)